db/init.sql

```diff
@@ -90,8 +90,120 @@ FROM (
 GROUP BY county, days_before
 ORDER BY days_before DESC;
 
-CREATE UNIQUE INDEX stats_by_county_index ON stats_by_county_day (county, days_before);
-CREATE INDEX stats_by_day_index ON stats_by_county_day (days_before);
+CREATE UNIQUE INDEX IF NOT EXISTS stats_by_county_index ON stats_by_county_day (county, days_before);
+CREATE INDEX IF NOT EXISTS stats_by_day_index ON stats_by_county_day (days_before);
+
+CREATE MATERIALIZED VIEW IF NOT EXISTS stats_by_county_day AS
```

Comment:
if i'm reading this right, it seems like we duplicated  this whole query here. is there a reason for that? more to the point: did we change anything from one to the other?

(i'm guessing this was probably just a harmless copy/paste error but i'm just trying to resolve some of my conflicts and wanted to make sure i didn't miss something here)

Reply:
yeah, i think this is just a copy/paste error, sorry!